server/src/controllers/christmas-tree.es6

```diff
@@ -393,30 +393,25 @@ const checkPermitValid = permitExpireDate => {
  * @function generateRulesAndEmail - Private function to generate svg, png, and rules html of a permit and send out email
  * @param {Object} permit - permit object
  */
-const generateRulesAndEmail = permit => {
+christmasTree.generateRulesAndEmail = permit =>
   permitSvgService
     .generatePermitSvg(permit)
-    .then(permitSvg => {
-      permitSvgService.generatePng(permitSvg).then(permitPng => {
-        permitSvgService
-          .generateRulesHtml(true, permit)
-          .then(rulesHtml => {
-            permit.permitUrl = paygov.createSuccessUrl(permit.christmasTreesForest.forestAbbr, permit.permitId);
-            let rulesText = htmlToText.fromString(rulesHtml, {
-              wordwrap: 130,
-              ignoreImage: true
-            });
-            sendEmail(permit, permitPng, rulesHtml, rulesText);
-          })
-          .catch(error => {
-            logger.error(error);
-          });
+    .then((permitSvg) => Promise.all([
+      permitSvgService.generatePng(permitSvg),
+      permitSvgService.generateRulesHtml(true, permit),
+    ]))
+    .then(([permitPng, rulesHtml]) => {
+      console.log('rules made');
+      permit.permitUrl = paygov.createSuccessUrl(permit.christmasTreesForest.forestAbbr, permit.permitId);
+      let rulesText = htmlToText.fromString(rulesHtml, {
```

Comment:
Should this be a `const` since it isn't changed?  It can give some context clues about whether you ever expect it to change.

Reply:
also you helped point out a lingering console log!